packages/suite/src/components/wallet/WalletLayout/AccountTopPanel/AccountNavigation.tsx

```diff
@@ -18,6 +19,8 @@ export const ACCOUNT_TABS = [
 ];
 
 export const AccountNavigation = () => {
+    const isDebugModeActive = useSelector(selectIsDebugModeActive);
```

Comment:
could you please add all those is debug things into 1 separate commit or add some specific TODOs somewhere so that we easily find what we should remove before launch?

Reply:
created a separate commit for this https://github.com/trezor/trezor-suite/pull/15409/commits/1a7595a4e0c8eb80d3963363c94f711bf5a4e3a4